pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1279,7 +1284,13 @@ func podToGroupMember(pod *v1.Pod, includeIP bool) *controlplane.GroupMember {
 		}
 	}
 	if includeIP {
-		memberPod.IPs = []controlplane.IPAddress{ipStrToIPAddress(pod.Status.PodIP)}
+		ips := sets.NewString(pod.Status.PodIP)
```

Comment:
curious why we need the set operation here. pod.Status.PodIPs won't have duplicates right?

Reply:
`pod.Status.PodIPs` should have some IP overriden with `pod.Status.PodIP`.

Reply:
PodIP must be same as PodIPs[0]: https://github.com/kubernetes/kubernetes/blob/f3fbd17bc8b4f82c4a49969b2b017afef19701d3/pkg/apis/core/v1/conversion.go#L283-L284
https://github.com/kubernetes/kubernetes/blob/f3fbd17bc8b4f82c4a49969b2b017afef19701d3/staging/src/k8s.io/api/core/v1/types.go#L3607-L3608
So we could just read PodIPs.

Reply:
I have updated code. `sets.NewString(pod.Status.PodIP) -> sets.NewString()`
Also, I think the UT `podToGroupMember` is wrong. In networkpolicy_controller_test.go, getPod() method should return a Pod with PodIPs, not just PodIP.

Reply:
I think the set is not needed here, it was used to remove duplicate IPs between `PodIP` and `PodIPs`, now we read only `PodIPs`, which is guaranteed to be at most one IPv4 IP and one IPv6 IP.
```
		for _, podIP := range pod.Status.PodIPs {
			memberPod.IPs = append(memberPod.IPs, ipStrToIPAddress(podIP))
		}
```

Reply:
Make sense. But actually `podIP.IP` is a string, so `ipStrToIPAddress(podIP) -> ipStrToIPAddress(podIP.IP)`